crates/telio-model/src/api_config.rs

```diff
@@ -303,6 +303,13 @@ where
     Ok(Some(eps))
 }
 
+/// Turns on the no link detection mechanism
+#[derive(Default, Copy, Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureNoLinkDetection {
+    /// Configurable rtt in seconds
+    pub rtt_seconds: Option<u64>,
```

Comment:
As in the previous node, I'm voting for using `#[serde(default = "...")]` here. It would save you a couple of `match`es down the road

Reply:
@dfetti what with this one?

Reply:
Done